### PR TITLE
LPS-195681 [Marketplace] Adjusted borders and margins of hero image

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 1]
```diff
@@ -25,6 +26,8 @@
 }
 
 .banner__contact-button {
+	border-radius: 8px !important;
+	padding: 12px 16px;
```
Comment (ytuik) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:30]: Can you use bootstrap for these properties?

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 2]
```diff
@@ -40,6 +43,8 @@
 }
 
 .banner__free-trial-button {
+	border-radius: 8px !important;
+	padding: 12px 16px;
```
Comment (ytuik) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:47]: Try use bootstrap for these properties

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 3]
```diff
@@ -96,9 +101,18 @@
 		align-items: flex-end;
 		align-self: center;
 		flex-direction: column;
+		gap: 0.5rem;
 		max-width: 200px;
```
Comment (ytuik) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:105]: Try use bootstrap for gap,
```suggestion 
    gap-3
```

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css [hunk 4]
```diff
@@ -96,9 +101,18 @@
...
 	}
 
+	.banner__contact-button {
+		padding: 10px 16px;
+	}
+
+	.banner__free-trial-button {
+		padding: 10px 16px;
+	}
```
Comment (ytuik) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/index.css:114]: try use bootstrap for it

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css [hunk 5]
```diff
@@ -7,19 +7,38 @@
...
+	position: absolute;
+	width: 100%;
 }
 
 .hero-container__image-hero {
-	margin: 0px auto;
-	margin-top: 6rem;
-	width: 80%;
+	margin-top: 23px;
+	position: relative;
```
Comment (ytuik) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css:17]: these two can be convertible on bootstrap

File: modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css [hunk 6]
```diff
@@ -7,19 +7,38 @@
 }
 
 .hero-container__image {
-	background-image: url('[resources:bg_hero.svg]');
-	background-repeat: no-repeat;
-	background-size: 100% 95%;
-	border-radius: 0.75rem;
+	height: 217px;
+	position: absolute;
+	width: 100%;
```
Comment (ytuik) [inline modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/hero/index.css:12]: these two can be convertible on bootstrap

```suggestion
	position-absolute and w-100
```

**Review by ytuik — CHANGES_REQUESTED** [review]
Try to change the new CSS variables in bootstrap

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**